inst/app/www/text/information.yml

```
@@ -1,56 +1,56 @@
 information:
   - image_src: "www/images/Person clipboard.jpg"
-    image_alt: "Person holding clipboard"
+    image_alt: " A surveyor undertaking data collection next to a waterbody"
```

In case the space is relevant.
```suggestion
    image_alt: "A surveyor undertaking data collection next to a waterbody"
```